vortex-array/src/array/sparse/compute.rs

```diff
@@ -46,6 +56,7 @@ impl AsContiguousFn for SparseArray {
                     .collect_vec(),
             )?,
             arrays.iter().map(|a| a.len()).sum(),
+            fill_types.first().unwrap().clone(),
```

Comment:
The API is that `self` == `arrays[0]`. It started out being a janky thing, but I actually quite like it because it means not everyone needs to assert that arrays.len > 0. 

Anyway, what this means is you can just write `self.fill_value().clone()`